md/release-notes.md

```diff
@@ -1,42 +1,107 @@
 # Release notes
------
-[ to remove at Beta time ]
-Don't forget to link your announcement to documentation pages that tell more about them.
-We have forgotten that in 7.5
------
+
 <a id="living-application-development-and-deployment"/>
 
-## Living Application development and deployment
+## Bonita BPM becomes Bonita
+Bonita's mission is to provide a platform allowing our customers to create highly tailored digital user experiences for their customers and employees. This is the reason why, while staying strongly BPM-based, the scope of the platform opens, and so is its name.
+Simply Bonita.
+
+## New values added
+
+### Industrialization of Living Application in Bonita Studio: Profiles
+Our goal is to provide you with a as few deployment artifacts as possible when creating applications.
+To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.
+After application descriptors, profiles are now available in Bonita Studio.
+In Community and Teamwork editions, you can edit the default profiles mapping with organization entities.
+In Efficiency and Performance editions, you can also create custom profiles, map them to the organization, and create their Bonita Portal menus if needed.
+Profiles are bundled in files of profiles, to allow to create a file of profiles per target environment / target organization.
+
+### Process display name
+It is now possible to add a display name for a process, so the development team can still use the technical "name" and provide a readable name for users. If not set, the technical name is used.
+
+### SAML logout
+SAML logout has been implemented, so that a user logging out of Bonita is also loggued out of all applications managed by the compagny Single-Sign-On system.
+
+### New API method to protect task assignation
+As an alternative to the current API method, a new method has been developped to prevent AssignHumanTasks to overwrite assignee programmatically. A configuration file allows the switch.
```

Comment:
I'm not sure how to understand what has been done, when reading this. Which config file are we talking about?

Reply:
Anyone who helped Pablo on this should be able to answer. I personally don't know but I agree this should be part of the article.
I'll ask Nico.

Reply:
This configuration file was a Spring Configuration made by Laurent I think. But I hope it is not the final fix we implemented in the product... :-/

Reply:
Can anyone with knowledge clarify this topic please? @laurentleseigneur maybe?